packages/paddlejs-mediapipe/camera/src/index.ts

```diff
@@ -189,18 +171,18 @@ export default class Camera {
 
     public start() {
         this.video && this.video.play();
-        if (this.requestAnimationId) {
```

Comment:
method  private videoRequestAnimationFrame() 
del line 187： this.requestAnimationId = window.requestAnimationFrame(drawImage);
use drawImage()

Reply:
fixed~